tests/integration/jobs/jobs.spec.ts

```diff
@@ -136,6 +136,124 @@ function createJobDataForGetJob(): unknown {
   return jobModel;
 }
 
+function isJobHasPendingTasksMock(tasks: TaskEntity[] | undefined): boolean {
+  if (tasks === undefined) {
+    return false;
+  }
+  return tasks.some((task) => task.status === OperationStatus.PENDING);
+}
+
+function createJobDataForAvailableActionsWithoutAbortableJob(): unknown {
```

Comment:
1. Consider extracting the mocked data creator to another file or function instead of multiplying the code.
2. Consider using random data instead of hard coded values. The tests will cover a lot more options. The library: '@ngneat/falso' does a good job at randomizing data :)

Reply:
Currently can't implement it.
We need to create separated task and focused on those changes in the future.